ui/components/TroubleshootingComponent.js

```diff
@@ -1,24 +1,86 @@
 import * as React from 'react';
 import Button from '@material-ui/core/Button';
 import TroubleshootingModal from './TroubleshootingModalComponent';
+import Modal from './Modal';
+import { helpAndSupportModalSchema, helpAndSupportModalUiSchema } from '@layer5/sistent';
+import { useNotification } from '@/utils/hooks/useNotification';
+import { useGetLoggedInUserQuery } from '@/rtk-query/user';
+import SupportIcon from '@/assets/icons/support';
+import axios from 'axios';
+import { EVENT_TYPES } from 'lib/event-types';
 
 const Troubleshoot = (props) => {
   const [open, setOpen] = React.useState(true);
+  const [openForm, setOpenForm] = React.useState(false);
+  const { notify } = useNotification();
   const handleOpen = () => {
     setOpen(true);
   };
 
+  const { data: userData } = useGetLoggedInUserQuery();
+
+  const handleOpenForm = () => {
+    setOpenForm(true);
+  };
+
+  const handleSupportFormClose = () => {
+    setOpenForm(false);
+  };
+
+  const handleSupportFormSubmission = async (data) => {
+    try {
+      await axios.post('https://hook.us1.make.com/r5qgpjel5tlhtyndcgjvkrdkoc65417y', {
```

Comment:
@sudhanshutech, thanks! Please note that this webhook cannot be included in Meshery directly, but must be in an extension.

Reply:
@yash37158 Will you add an API in meshery server that proxies this support request to remote provider and then the provider (in the backend) handles this?

Reply:
@MUzairS15 Yeah sure

Reply:
@Yashsharma1911 ^^

Reply:
@sudhanshutech ?